src/utils.ts

```diff
@@ -89,7 +94,18 @@ export const getCodeRef = (
 };
 
 export const sendEventToReporter = (type: string, data: any, suite?: string): void => {
-  process.stdout.write(JSON.stringify({ type, data, suite }));
+  const annotation = {
+    type: type,
+    description: JSON.stringify(data),
+  };
+  if (suite) {
+    if (!RPReporter.sharedSuitesAnnotations.get(suite)) {
+      RPReporter.sharedSuitesAnnotations.set(suite, []);
+    }
+    RPReporter.sharedSuitesAnnotations.get(suite).push(annotation);
```

Comment:
This approach won't work while adding suite data from the tests.
So let's discuss the alternative on the call.